code/game/objects/items/weapons/grenades/chem_grenade.dm

```diff
@@ -10,6 +10,8 @@
 	var/path = 0
 	var/obj/item/device/assembly_holder/detonator = null
 	var/list/beakers = new/list()
+	var/list/temp_reagents = new/list()
```

Comment:
No, it was actually just because I thought making a new one down in the for loop would be frowned up.
The `reagents_text` var can't be made any further in than just above the for loop. Is it alright to put it there or should that still be at the top?